Changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.5.4 - (2021-07-29)
```

Comment:
Since we are adding a new feature, we should increment the minor version instead.